lips/evaluation/airfrans_evaluation.py

```diff
@@ -123,14 +124,20 @@ def evaluate_ml(self) -> dict:
         """
         self.logger.info("Evaluate machine learning metrics")
         #Normalize prediction and observation before computing mlMetrics
-        chunk_sizes = [int(simulation[1]) for simulation in self.observation_metadata["simulation_names"]]
 
         field_names = self.predictions.keys()
-        flattened_observation = np.concatenate([self.observations[field_name][:, None] for field_name in field_names], axis = 1)
-        mean_observ,std_observ = iterative_fit(flattened_observation,chunk_sizes)
+        if self.ml_normalization is None:
+            self.logger.info("Using current dataset to normalize")
+            chunk_sizes = [int(simulation[1]) for simulation in self.observation_metadata["simulation_names"]]
+            flattened_observation = np.concatenate([self.observations[field_name][:, None] for field_name in field_names], axis = 1)
```

Comment:
I think the `self.observations` here, refers to the ground truth test dataset passed through the evaluation call. However, as your aim is to fit the scaler parameters on train dataset and use them to normalize the test dataset, I think this does not make sense.